src/scss/component/accordion/_index.scss

```diff
@@ -0,0 +1,111 @@
+// @gruapl/graupl accordion component styles.
+
+@use "defaults";
+@use "variables" as *;
+@use "../../defaults" as root-defaults;
+@use "../../mixins/layer" as *;
+
+#{defaults.$accordion-selector} {
+  @include layer(component) {
+    border-radius: #{$accordion-border-radius};
+    border: #{$accordion-border};
+
+    #{defaults.$accordion-item-selector} {
+      &>*:first-child {
```

Comment:
This is a super rigid as far as structure would go. Why first-child? What if I have something above the toggle in the markup?

Reply:
I added a class for the element I wanted to select. The h3 is adding margin, and I am assuming that the h3 could be something else. I added a class to the h3 so I can select it better.